pytorch_lightning/trainer/deprecated_api.py

```diff
@@ -87,3 +87,20 @@ def nb_sanity_val_steps(self, nb):
                       "`num_sanity_val_steps` since v0.5.0"
                       " and this method will be removed in v0.8.0", DeprecationWarning)
         self.num_sanity_val_steps = nb
+
+    @property
+    def use_amp(self):
+        """Back compatibility, will be removed in v0.9.0"""
+        warnings.warn("`use_amp` has been deprecated in favor of `precision` since v0.7.0"
+                      " and will be removed in v0.9.0", DeprecationWarning)
+        return self.precision == 16
+
+    @use_amp.setter
+    def use_amp(self, use_amp):
```

Comment:
as this is deprecated until v0.9 and the class is v0.8 you need to create a new mixin

Reply:
Ah yes of course. Should this be in a new file, or in the same `deprecated_api.py` file?

Reply:
just a new class in the file is fine :]